Task_objects/index.js

```diff
@@ -1,24 +1,45 @@
-const cat = {
```

Comment:
The 'const' keyword indicates that the variable is a constant and cannot be reassigned. Since this object is mutable and its properties can change, using 'let' instead of 'const' may be more appropriate here.